### DISCUSSION TITLE
I encountered a situation with the firewall settings on Windows where the device can send files but cannot receive files or be discovered. Here is my solution

### DISCUSSION THREAD

**Discussion by wakizashi1024** [localsend/localsend#801] [Discussion (English, International)]
My environment:
- A MacBook Pro 2018 with MacOS Ventura
- A Windows laptop with Windows 11

Windows send files to MacOS: success
MacOS send files to Windows: failed(timeout)

After I did this tries, the problem fixed.
Step 1. Add firewall inbound tcp/udp rules(choose one from auto fix and manual) => still not working
Step 2. Go to Settings => Privacy & security => Windows Security => Firewall & network protection
![Screenshot 2023-10-25 135721](https://github.com/localsend/localsend/assets/100260186/b0491ec2-c999-4f23-bf62-b572a533270b)
Step 3. Choose your active networking(my active network is "Public network")
![Screenshot 2023-10-25 135806-1](https://github.com/localsend/localsend/assets/100260186/760d5a34-ef29-4aec-b995-169766323edb)
Step 4. On topic "Incoming connections", disable the option "Blocks all incoming connections, including those in the list of allowed apps.
![Screenshot 2023-10-25 135918](https://github.com/localsend/localsend/assets/100260186/e2f151a1-7540-46c4-9af8-5e39c14e27da)

**Comment by fscheps** [comment]
Hi there, thanks for sharing, if you are doing this when you are in a local trusted network, something seems off no?
